### ISSUE TITLE
Listen to more pull_request actions?

### ISSUE THREAD

**Issue by gr2m** [hiimbex/markdownify#3]
Hey @hiimbex :wave: 

right now the app only listens to the "open" action

https://github.com/hiimbex/markdownify/blob/f83012eafcb38b12de61e55e18305359dccb92fc/index.js#L2

After installing the app, I’d expect that the app would also listen to the "synchronize" action as a markdown file could be added / removed over time in a pull request?

Happy to send a PR if you’d like that

**Comment by ChrisMM** [comment]
@hiimbex @gr2m I think this is the same issue, but admittedly I'm not as technical:

**Steps**
1. Created a PR that had one new .md file changed (added) in the branch
2. Updated a README.md to point to newly-created .md file

**Expected**
Updated README.md would be added to "View rendered" list in PR description

**Actual**
Updated README.md is not added in PR description in "View rendered" list; only one new .md file changed (added) in the branch is shown

**Comment by hiimbex** [comment]
Yup! That's exactly the issue @ChrisMM. And ideally we'd filter out linked to files that are completely removed as well; right now we're only looking for edited/changed which could include an entire removal.
